client/src/components/Multiselect.tsx

```diff
@@ -0,0 +1,735 @@
+// This component is slightly adapted from https://github.com/srigar/multiselect-react-dropdown
```

Comment:
hmm wondering if it would be worth importing and extending from their component instead of forking, then overwriting the functions we need (render, etc)? would be able to quickly pull future changes/fixes to this, but might be more effort than it's worth

Reply:
yeah, I think you're right, that would be better. I ended up needed to change a fair bit across multiple methods, but still better to do it that way. I'll plan to do come back and do this soon.